### ISSUE TITLE
Not tesselating!

### ISSUE THREAD

**Issue by jredfox** [TheWeatherPony/TheSeasonsMod#15]
version: beta8b and  beta8c3?
http://pastebin.com/7C4SB1Jm

**Comment by TheWeatherPony** [comment]
Your trace only shows half of a problem, and doesn't link the cause on either end to anything I cause. My code shows up in the trace only because it has to in order for the rest of the code to run with my changes - my changes are not causing this to happen in what you have shown.

I am leaving this open in case you can get any other traces for this that might link back to my code more than circumstantially.

**Comment by jredfox** [comment]
Ok I am unsure if this was caused by your mod the launcher or both since this was a previous mc launcher release. 2 days ago I also had a crash like this but, it was just vanilla when I launched mc.  With better foliage the same thing would happen already teesalting error if you went withing 15 blocks of a loading chunk. That occured before they updated the launcher so I think it is both the launcher and the mod. However if they fixed the mc launcher there should be no more probolems.  same for all teesalating errors.
